lib/run_argument_parser.py

```diff
@@ -42,6 +42,12 @@ def __init__(self, tool_name, input_file_help):
                  + "Typical epitope lengths vary between 8-11. "
                  + "Required for Class I prediction algorithms.",
         )
+        parser.add_argument(
+            "--species",
+            help="The species of the test subject(s). Will be used to check allele validity and for reference proteome search.",
+            default="human",
+            choices=['human', 'mouse', 'chimpanzee', 'macaque', 'cow', 'horse', 'pig'],
+        )
```

Comment:
Are the alleles themselves insufficient to infer the species?

Reply:
This is to prevent a user from using a combination of alleles from different species. .

Reply:
I guess what I'm asking is, does it make sense to simplify the inputs? Is it possible to just require the alleles, infer the species (and we can still check to make sure there is not more than one species represented and throw an error).  Seems like that would be a usability win - one less place for the user to screw up an input - but would it be a big hurdle in the code?

Reply:
I pushed a commit to infer the species from the alleles provided and error out if there is a mismatch.